jekyll/_cci2/android-machine-image.md

```diff
@@ -199,3 +194,62 @@ workflows:
 ```
 {% endraw %}
 
+### Using the android image on server v3.x
+{: #using-the-android-image-on-server-v3x }
+
+**Note**: Android machine images are only available on server installations on Google Cloud Platform (GCP) at this time.
+
+From CircleCI server 3.4, Android machine images are supported for installations on GCP. To use the Android image in your projects set the `image` key to `android-default` in your jobs.
+
+```yaml
+version: 2.1
+
+jobs:
+  my-job:
+    machine:
+      image: android-default
+    steps:
+    # job steps here
+```
+
+It is also possible to use the android orb, as shown above for cloud. Your server administrator will need to import the orb first. Also, you will need to define the `android-default` image for the machine executor, as shown in the example below, rather than using the default executor built into the orb. View the [CircleCI Server v3.x Orbs](https://circleci.com/docs/2.0/server-3-operator-orbs) page for instructions on importing orbs.
```

Comment:
```suggestion
It is also possible to use the android orb, as shown above, for cloud. Your server administrator will need to import the orb first. Also, you will need to define the `android-default` image for the machine executor, as shown in the example below, rather than using the default executor built into the orb. View the [CircleCI Server v3.x Orbs](https://circleci.com/docs/2.0/server-3-operator-orbs) page for instructions on importing orbs.
```